packages/react/src/redux.ts

```diff
@@ -89,6 +96,17 @@ function createReduxEnhancer(enhancerOptions?: Partial<SentryEnhancerOptions>):
 
   return (next: StoreEnhancerStoreCreator): StoreEnhancerStoreCreator =>
     <S = any, A extends Action = AnyAction>(reducer: Reducer<S, A>, initialState?: PreloadedState<S>) => {
+      options.attachReduxState &&
+      event.contexts && 
+      event.contexts.state &&
+        addGlobalEventProcessor((event, hint) => {
+          hint.attachments = [
+            ...(hint.attachments || []),
+            { filename: 'reduxState.json', data: JSON.stringify(event.contexts.state) },
```

Comment:
I prefer just stringifying the context value as `type: 'redux'` is redundant.

let's also use lower casing for the filename.

```suggestion
            { filename: 'redux_state.json', data: JSON.stringify(event.contexts.state.value) },
```